lib/jira/client/axios.js

```diff
@@ -43,14 +42,6 @@ function getErrorMiddleware (id, installationId) {
       params: error.config.fields
     }, `Jira request: ${error.request.method} ${error.request.path} - ${status} ${statusText}`)
 
-    Sentry.configureScope(scope => {
```

Comment:
This is the bit that set context globally. Once this ran, any exception that came after it was tagged with this context.